lib/types.ts

```diff
@@ -39,6 +39,7 @@ export interface Highlights {
   pr_collaborated: number;
   issue_assigned: number;
   issue_opened: number;
+  newContributor?: boolean;
```

Comment:
Also, move this field to the "Contributor" type instead of keeping it under "Highlights", since it's specific to a contributor. Highlights are used in multiple other ways where field `newContributor` wouldn't be a right fit.